app/js/components/obs.jsx

```diff
@@ -446,29 +337,41 @@ export default class Obs extends React.Component {
         toastr.info('Nothing to update');
     }
   }
-  /**
-   * render - renders the component
-   * 
-   * @returns 
-   * 
-   * @memberOf Obs
-   */
+
   render() {
     return (
       <div>
         {
-          this.state.isShowingModal &&
-          <ModalContainer onClose={this.handleClose}>
-            <ModalDialog onClose={this.handleClose}>
-              <h1>Observation Dialog</h1>
-              <p>Are you sure you want to delete? <br />
-                Click yes to delete or close this modal to cancel</p>
-              <div>
-                <button type="button" name="modalDelete" onClick={this.delete}
-                className="btn btn-default form-control cancelBtn">Yes</button>
+          <div className="modal fade" id="myModal" role="dialog">
+          <div className="modal-dialog">
+            <div className="modal-content">
+              <div className="modal-header">
+                <button type="button" className="close" data-dismiss="modal">&times;</button>
+                <h4 className="modal-title">Confirm Delete</h4>
+              </div>
+              <div className="modal-body">
+                <p> Are you sure want to Delete this Observation?</p>
               </div>
-            </ModalDialog>
-          </ModalContainer>
+              <div className="modal-footer">
+                <div className="col-sm-6">
+                  <button
+                    type="button" className="btn btn-success form-control"
+                    onClick={this.delete}
+                    data-dismiss="modal"
+                  >Confirm
+                  </button>
+                </div>
+                <div className="col-sm-6">
+                  <button
+                    type="button" className="btn btn-danger form-control cancelBtn"
+                    data-dismiss="modal"
+                  >Close
```

Comment:
Looks like a tab is missing here

Reply:
Kindly view the file, the indentation is two white space as specified here https://github.com/felixge/node-style-guide#2-spaces-for-indentation referenced https://wiki.openmrs.org/display/docs/JavaScript+Conventions